src/calcium/qqbar.jl

```diff
@@ -180,6 +180,9 @@ zero(a::CalciumQQBarField) = a(0)
 
 one(a::CalciumQQBarField) = a(1)
 
+# Exists only to support Julia functionality (no guarantees)
+zero(::Type{qqbar}) = CalciumQQBar(0)
```

Comment:
Since `CalciumQQBarField` is a singleton type, this is safe, so I don't think we even need that "no guarantees" comment?

For symmetry, perhaps also add `one(::Type{qqbar}) = CalciumQQBar(1)`?

Finally, a test case would be nice (though no requirement from my POV)

Reply:
Removed the comment.
Added `one(qqbar)`.
Did similar things for `QQFieldElem` and `ZZRingElem`.

Reply:
Awesome, thanks

Reply:
Wonderful, thank you!